src/ts/ConvivaAnalytics.ts

```diff
@@ -1,5 +1,5 @@
 import * as Conviva from '@convivainc/conviva-js-coresdk';
-import type {
+import {
```

Comment:
You're right, but it's not TS related in this case, but webpack, since webpack bundles the files in this repo. Here is a fix https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/115/commits/fda346eba962b2411e222bd48b70de119aa0c6ed.